supportedBackends/htcondor/src/main/scala/cromwell/htcondorbackend/HtCondorBackend.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.htcondorbackend
```

Comment:
Yeah I think you're right that would be confusing.  Maybe something like `cromwell.backend.impl.<backendname>`?